spec/index.bs

```diff
@@ -5869,33 +5873,40 @@ dictionary GPUShaderModuleDescriptor : GPUObjectDescriptorBase {
             <div data-timeline=device>
                 [=Device timeline=] |initialization steps|:
 
-                1. If any of the following conditions are unsatisfied
-                    [$generate a validation error$], make |sm| [=invalid=], and stop.
+                1. Let |result| be the result [=shader module creation=] with the WGSL source
+                    |descriptor|.{{GPUShaderModuleDescriptor/code}}.
+                1. If any of the following requirements are unmet,
+                    [$generate a validation error$], make |sm| [=invalid=], and return.
 
                     <div class=validusage>
-                        - |this| is [=valid=].
+                        - |this| must be [=valid=].
+                        - |result| must not be a [=program error=].
```

Comment:
Better to use [=shader-creation error=] here

https://gpuweb.github.io/gpuweb/wgsl/#shader-creation-error 

It's the set of errors that can (and should) be detected at createShaderModule time.


At lot of the "must" statements in the WGSL spec are links to [=shader-creation-error=]

Reply:
I wanted to avoid capturing "uncategorized errors" here. Does "shader-creation error" exclude those?

Reply:
@dneto0 PTAL